pages/1.11/release-notes/1.11.0-rc1/index.md

```diff
@@ -86,14 +86,14 @@ Please try out the new features and updated data services. Provide any feedback
 - Non-essential tasks now supported for DC/OS Kafka, DC/OS Cassandra, DC/OS Elastic, and DC/OS HDFS. If non-essential tasks fail, the entire node will not fail.
 
 # <a name="known-issues"></a>Known Issues and Limitations
-- Upgrades from 1.10 to 1.11 are _not supported_ in 1.11 Release Candidate 1.
+- Upgrades from 1.10 to 1.11 are _not supported_ in 1.11.0 Release Candidate 1.
+- DCOS-19047 - `dcos-secrets` service is unavailable during upgrade from 1.10.x to 1.11.0. 
 
-- DCOS-19047 - dcos-secrets service is unavailabe during upgrade from 1.10.x to 1.11. 
 
-# <a name="fixed-issues"></a>Improvements and Major Issues Fixed in 1.11 Release Candidate 1
+# <a name="fixed-issues"></a>Improvements and Major Issues Fixed in 1.11.0 Release Candidate 1
 - DCOS-16368 -  DC/OS Enterprise Secret Store no longer accepts secrets with uppercase letters in their paths. Secret paths with uppercase letters conflicted with use of the IAM API. [enterprise type="inline" size="small" /]
```

Comment:
This change has been reverted: https://github.com/mesosphere/dcos-secrets/pull/63